### ISSUE TITLE
HKSV recording for AI Theta fails starting in 6.9.0

### ISSUE THREAD

**Issue by reznorts** [hjdhjd/homebridge-unifi-protect#958]
**Describe The Problem:**

HKSV recording from an AI Theta camera connected to a UDM-SE works under 6.8.1 but fails under 6.9.0

**To Reproduce:**

Configure plugin for HKSV event recording and perform a test, this works. After upgrading to 6.9.0 HKSV recording no longer occurs.

**Logs:**

Logging under 6.8.1 shows:
[7/6/2023, 8:12:56 PM] [UniFi Protect] UDM SE [UDM-PRO-SE] Living room [AI Theta]: HomeKit Secure Video hardware accelerated event recording enabled: 1280x1280@20fps (Medium), 2000 kbps with a 4 second timeshift buffer.

followed by this when an event occurs:
[7/6/2023, 8:13:43 PM] [UniFi Protect] UDM SE [UDM-PRO-SE] Living room [AI Theta]: HomeKit Secure Video has recorded a 18 second motion event.

Under 6.9.0 I see other cameras (G5 Flex) being enabled for event recording but the AI Theta camera is not listed in the logging.

**Homebridge Configuration:**

```json
{
"controllers": [
{
"address": "192.168.10.1",
"password": "redacted",
"username": "redacted"
}
],
"name": "UniFi Protect",
"options": [
"Disable.Audio.AC8BA90D97BB",
"Enable.Video.Transcode.Hardware.60223223EE1D",
"Enable.Audio.Filter.Noise.60223223EE1D",
"Enable.Audio.Filter.Noise.E438830EF0EF",
"Enable.Audio.Filter.Noise.E438830EF1A7",
"Enable.Video.Transcode.Hardware.E438830EF1A7",
"Enable.Video.Transcode.Hardware.E438830EF0EF",
"Enable.Video.Transcode.Hardware.AC8BA90D97BB",
"Enable.Video.Transcode.Hardware.E438830EF114",
"Enable.Audio.Filter.Noise.E438830EF114",
"Disable.Log.Doorbell.AC8BA90D97BB",
"Enable.Video.Stream.Only.Medium.60223223EE1D",
"Disable.Video.Stream.Only.Medium.E438830EF1A7",
"Enable.Video.Stream.Only.Medium.AC8BA90D97BB"
],
"verboseFfmpeg": false,
"_bridge": {
"username": "redacted",
"port": 47757
},
"platform": "UniFi Protect"
}
```

**Screenshots:**

**Environment:**

* **Homebridge Version:**  1.6.1
* **Node Version:**  18.16.1
* **Homebridge-UniFi-Protect Plugin Version:** 6.9.0
* **UniFi Protect Version:**  2.8.28
* **UniFi Protect Controller Type:**  UDM-SE
* **UniFi Protect Controller Firmware Version:**  3.0.18
* **Apple Device and iOS / macOS / iPadOS / tvOS Version:** macOS 13.4.1 (22F82) & iOS 16.5.1 (20F75)
* **Operating System and OS Version:**  macOS 13.4.1 (22F82)

**Comment by hjdhjd** [comment]
Thanks for the report. Let me see if I can replicate this issue. Can you map the MAC addresses associated with the configuration options with the respective cameras and Protect controllers? I can’t tell which options apply to which camera/controller.

**Comment by reznorts** [comment]
AI Theta  ac:8b:a9:0d:97:bb
G5 Flex #1 e4:38:83:0e:f1:a7
G5 Flex #2 e4:38:83:0e:f1:14
G5 Felx #3 e4:38:83:0e:f0:ef
UDM SE 60:22:32:23:ee:1d

Let me know if you need anything more.

**Comment by hjdhjd** [comment]
Will be fixed in the next release. Thanks for raising this...and sorry about that!

**Comment by reznorts** [comment]
Thank you!

**Comment by hjdhjd** [comment]
Fixed in v6.10.0 - enjoy!

**Comment by reznorts** [comment]
Updated to 6.10.0 and confirmed fix.
